Sources/StreamChatUI/Controller/ChatGroupDetailsVC.swift

```diff
@@ -13,84 +13,71 @@ import UIKit
 
 public class ChatGroupDetailsVC: ChatBaseVC {
     enum GroupDetailsSection: CaseIterable {
-        case userList,attachmentList
+        case userList
     }
     // MARK: - OUTLETS
     @IBOutlet private var lblTitle: UILabel!
     @IBOutlet private var lblSubtitle: UILabel!
     @IBOutlet private var tableView: UITableView!
     @IBOutlet private var notificationSwitch: UISwitch!
-    var filesContainerView = UIView()
     // MARK: - VARIABLES
     public var groupInviteLink: String?
     public var selectedUsers: [ChatChannelMember] = []
     public var channelController: ChatChannelController?
-    private var arrController = [UIViewController]()
-    private var usersCount = 0
-    private var sectionWiseList = [GroupDetailsSection]()
-    private lazy var buttonShowMore: UIButton = {
-        let btnShowMore = UIButton(frame: CGRect.init(x: UIScreen.main.bounds.width - 120, y: 0, width: 100, height: 35))
-        btnShowMore.setTitle("Show more", for: .normal)
-        btnShowMore.setTitleColor(Appearance.default.colorPalette.statusColorBlue, for: .normal)
-        btnShowMore.titleLabel?.font = UIFont.systemFont(ofSize: 14)
-        btnShowMore.addTarget(self, action: #selector(self.showMoreButtonAction(_ :)), for: .touchUpInside)
-        return btnShowMore
-    }()
+    private let sectionWiseList = GroupDetailsSection.allCases
     // MARK: - VIEW CYCLE
     public override func viewDidLoad() {
         super.viewDidLoad()
         setupUI()
+        fetchGroupDetails()
     }
     // MARK: - METHOD
     open func setupUI() {
```

Comment:
@jitendra-simform  Please make it private as it doesn't use outside of class.
Additionally, please check other variables and functions access control.